app/services/claims/generate_claims_csv.rb

```diff
@@ -0,0 +1,17 @@
+require "csv"
+
+class Claims::GenerateClaimsCsv
+  include ServicePattern
+
+  HEADERS = %w[school_name school_urn amount claim_reference_number].freeze
+
+  def call
+    CSV.generate(headers: true) do |csv|
+      csv << HEADERS
+
+      Claim.find_each do |claim|
+        csv << [claim.school.name, claim.school.urn, claim.amount.format(symbol: false, decimal_mark: ".", no_cents: false), claim.reference]
```

Comment:
Minor readability nitpick. This line's getting a bit long now. It's absolutely not a deal-breaker, but @CatalinVoineag might appreciate it as a Vim/Emacs(??) user.

```suggestion
        csv << [
          claim.school.name,
          claim.school.urn,
          claim.amount.format(symbol: false, decimal_mark: ".", no_cents: false),
          claim.reference,
        ]
```

Reply:
Neovim user to be specific 😄 :neckbeard: